### PR TITLE
Raise projects to 9.4

### PR DESCRIPTION
**Pull request by alexsuter**

### CODE REVIEW

File: doc-factory-test/cms/cms_en.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ivy-cst) [inline doc-factory-test/cms/cms_en.yaml:1]: why?

File: doc-factory/cms/cms_de.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+ch:
```
Comment (ivy-cst) [inline doc-factory/cms/cms_de.yaml:1]: A lot of \r\n are they needed?

**Review by ivy-cst — APPROVED** [review]
:+1:

File: doc-factory-test/cms/cms_en.yaml (hunk 1 above)
Comment (alexsuter) [reply doc-factory-test/cms/cms_en.yaml:1]: This was a bug. Fixed.

File: doc-factory/cms/cms_de.yaml (hunk 2 above)
Comment (alexsuter) [reply doc-factory/cms/cms_de.yaml:1]: As soon as I write a string to yaml with \r\n . SnakeYaml will make double quotes and keep \r\n.
Maybe SnakeYaml won't mixup the file with differente line endings, which would make sense to me.

What do you think? Should I transform all these \r\n to \n ?

Comment (ivy-cst) [reply doc-factory/cms/cms_de.yaml:1]: I think normally this was html, so it does not matter if it is \n or \r\n. To have a nicer yaml i would convert them I think...

**Pull request merged**